roles/sqs_queue/files/queue_attributes.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Where do you render this template?

Reply:
see https://gitlab.service.opg.digital/opsforks/opg-core-deploy/blob/OPGOPS-3131/ansible/build1/env_vars.yml